tests/test_setget_field.f90

```diff
@@ -28,7 +28,7 @@ program test_setget_field
 #endif
   type(mesh_t) :: mesh
   
-  class(field_t), pointer :: fld
+  class(field_t), pointer :: fld, fld_c
   real(dp), dimension(:, :, :), allocatable :: arr
 
   integer :: nx, ny, nz
```

Comment:
Also suprised that this doesn't throw a unused variable warning and tests are passing

Reply:
Done